### PR TITLE
Remove redundant shipment helper

### PR DESCRIPTION
**Pull request by mbj**
Extraction from #398

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: backend/app/views/spree/admin/orders/_line_items.html.erb [hunk 1]
```diff
@@ -42,7 +42,7 @@
             <% end %>
           </td>
           <td class="line-item-total align-center">
-            <%= line_item_shipment_price(item, item.quantity) %>
+            <%= Spree::Money.new(item.amount, currency: item.currency) %>
```
Comment (dkubb) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: You could probably replace all this with `<%= item.display_amount %>`

Comment (mbj) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: good catch. mid term I do want something more structured than having an inconsistent mix of `display_*` methods directly on the models.

Comment (dkubb) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: > mid term I do want something more structured than having an inconsistent mix of display_* methods directly on the models.

I could see us splitting out the `display_*` methods into a presenter layer and giving them the same name as the methods they proxy.

Comment (mbj) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: Yeah, this would also prevent computation on these in the view layer.

**Comment by mbj** [comment]
@dkubb ready for the next pass

**Pull request merged**